services/ui-src/src/routes/ReviewSubmit/ReviewSubmit.tsx

```diff
@@ -1,29 +1,215 @@
+import { useContext, useEffect } from "react";
 // components
-import { Box, Heading, Text } from "@chakra-ui/react";
-import { ReportPage } from "components";
-import verbiage from "verbiage/pages/mcpar/mcpar-review-and-submit";
+import {
+  Box,
+  Button,
+  Flex,
+  Image,
+  Heading,
+  Text,
+  useDisclosure,
+} from "@chakra-ui/react";
+import { Icon, Modal, ReportContext, ReportPage, Sidebar } from "components";
+// types
+import { ReportStatus, UserRoles } from "types";
+// utils
+import { useUser } from "utils";
+// form data
+import reviewVerbiage from "verbiage/pages/mcpar/mcpar-review-and-submit";
+import successVerbiage from "verbiage/pages/mcpar/mcpar-successful-submit";
+// assets
+import checkIcon from "assets/icons/icon_check_circle.png";
 
 export const ReviewSubmit = () => {
-  const { intro } = verbiage;
+  const { reportStatus, fetchReportStatus, updateReportStatus } =
+    useContext(ReportContext);
+  const { isOpen, onOpen, onClose } = useDisclosure();
+
+  // get user's state
+  const { user } = useUser();
+  const { state, userRole } = user ?? {};
+  const reportYear = "2022";
+  const reportKey = `${state}${reportYear}`;
+
+  // TODO: get real program name per report
+  const programName = "tempName";
```

Comment:
This will be changing and we'll get it from the ReportContext but this is fine for now.